### DISCUSSION TITLE
2 steppers 1 axis

### DISCUSSION THREAD

**Discussion by Miljnor** [arkypita/LaserGRBL#1803] [General]
I am using laserGRBL and currently have just 1 stepper on the X axis but the second rail on my axis is sticking periodicaly. So I was wondering if there was a thread I missed (searched and could find) or if there was a way to install another stepper with my spare axis and use them both synced together? 

Or some other way.

thanks
Michael

**Comment by StuartB4** [comment]
You mean the Y axis, X is the one with the laser on. 
Depends on the main control board you have.
If it has 4 sockets for the motors they will be X, Y1, Y2 and Z.
If it has 3 they could be X, Y1, Y2 but could also be X, Y, Z.

**Reply by Miljnor** [reply]
need 2 steppers on x axis and 1 on y axis.  cnc shield has 4 spots for drivers can laserGRBL slave 2 as 1 axis?

**Comment by StuartB4** [comment]
I don't understand why you need 2 steppers on the X axis, surely there is just 1 for the laser to move backward and forward. 

Do you use an Arduino with a protoneer CNC Shield.
If you do then you can use the pins on the left above the power connector marked X.

![shield](https://user-images.githubusercontent.com/16980026/161408793-e8bb9e5e-446c-4ea8-a5e7-1ed8eb6e80be.png)

**Reply by Miljnor** [reply]
perfect that is exactly what i was looking for!

thanks for your help!

**Reply by StuartB4** [reply]
You're welcome. Good luck with it. 👍
